### PR TITLE
Update CODEOWNERS and team names in TODO comments

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Documentation Update

## Description

This PR

1. updates the `CODEOWNERS`  file to reflect the move to `forem/forem`,
2. changes all TODO comments to the new team names and
3. standardizes the TODO comment format to `@forem/teamname`.
,
It seems we no longer have a "Team Cool", please advice who the `CODEOWNERS` entries and TODO comment should be assigned to.

## Related Tickets & Documents\

n/a

## Added tests?

- [X] no, because they aren't needed

## Added to documentation?

- [X] no documentation needed

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,36 +2,36 @@
...
+/*.md                                               @forem/oss
+/app/assets/stylesheets/internal/                   @forem/snackcase
+/app/controllers/articles_controller.rb             @forem/hells-bells
+/app/controllers/internal/                          @forem/snackcase
+/app/services/notifications/                        @forem/delightful
+/app/controllers/notifications/                     @forem/delightful
+/app/workers/notifications/                         @forem/delightful
+/app/controllers/internal/broadcasts_controller.rb  @forem/delightful
+/app/controllers/async_info_controller.rb           @forem/sre
+/app/controllers/pages_controller.rb                @forem/cool
```
Comment (citizen428) [inline .github/CODEOWNERS:14]: @forem/cool no longer exists, who should this be assigned to?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,36 +2,36 @@
...
+/app/controllers/stories_controller.rb              @forem/hells-bells
+/app/services/search/                               @forem/sre
+/app/views/articles/                                @forem/hells-bells
+/app/views/internal/                                @forem/snackcase
+/app/views/moderation/                              @forem/snackcase
+/app/views/notifications/                           @forem/delightful
+/app/views/partnerships/                            @forem/cool
```
Comment (citizen428) [inline .github/CODEOWNERS:21]: @forem/cool no longer exists, who should this be assigned to?

File: app/models/display_ad.rb [hunk 3]
```diff
@@ -27,7 +27,7 @@ def process_markdown
     markdown = Redcarpet::Markdown.new(renderer)
     initial_html = markdown.render(body_markdown)
     # Temporarily disable the sanitisation in order to launch the SheCoded Campaign.
-    # TODO: [@thepracticaldev/cool] find an alternate solution.
+    # TODO: [@forem/cool] find an alternate solution.
```
Comment (citizen428) [inline app/models/display_ad.rb:30]: @forem/cool no longer exists, who should this be assigned to?

File: .github/CODEOWNERS (hunk 1 above)
Comment (rhymes) [reply .github/CODEOWNERS:14]: I think no one. I would remove this line

File: .github/CODEOWNERS (hunk 2 above)
Comment (rhymes) [reply .github/CODEOWNERS:21]: I think no one. I would remove this line

File: app/models/display_ad.rb (hunk 3 above)
Comment (rhymes) [reply app/models/display_ad.rb:30]: I think no one. I would remove the parenthesis in the `TODO`

**Comment by citizen428** [comment]
Thanks, @rhymes! I ended up just removing the references to team Cool, sometimes the obvious solution is too easily forgotten 😃

**Pull request merged**